android/src/main/java/com/bitmovin/player/reactnative/RNPlayerViewManager.kt

```diff
@@ -99,9 +124,16 @@ class RNPlayerViewManager(private val context: ReactApplicationContext) : Simple
      * @param view Target `RNPlayerView`.
      * @param playerId `Player` instance id inside `PlayerModule`'s registry.
      */
-    private fun attachPlayer(view: RNPlayerView, playerId: NativeId?) {
+    private fun attachPlayer(view: RNPlayerView, playerId: NativeId?, playerConfig: ReadableMap?) {
         Handler(Looper.getMainLooper()).post {
             val player = getPlayerModule()?.getPlayer(playerId)
+            // set picture in picture value based on playback config's option
+            playerConfig?.getMap("playbackConfig")?.getBoolean("isPictureInPictureEnabled")?.let {
+                // set PiP handler's `isPictureInPictureEnabled` value.
+                pictureInPictureHandler.isPictureInPictureEnabled = it
+                // set the view's PiP handler instance.
```

Comment:
Comment not needed

Reply:
Done 8da9939266acb2fc41cd337092888c511d6bcefb